doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile

```diff
@@ -3,4 +3,5 @@ h1. Python4Capella User Manual
 This document describes how to use Python4Capella.
 
 You will find the following sections:
```

Comment:
I think that we need two different sections in the developer guide:
* Advanced Python users: with topics like "Install new Python packages"
* Python4Capella contributors: with topics like "Create a Python interpreter distribution for Windows"